base/src/main/java/pl/edu/icm/unity/base/msgtemplates/confirm/EmailConfirmationTemplateDef.java

```diff
@@ -51,7 +51,7 @@ public String getName()
 	@Override
 	public Set<String> getCompatibleFacilities()
 	{
-		return Stream.of(FacilityName.EMAIL.toString())
-	                        .collect(Collectors.toCollection(HashSet::new));
+		return Collections.unmodifiableSet(Sets
```

Comment:
Not very needed to fix it (not a bug) but here ensuring that ad-hoc created collection is read only is rather not needed. User can do anything with it :-)